### ISSUE TITLE
[Feature Request] better visibility for failed / error feeds

### ISSUE THREAD

**Issue by aslmx** [SSilence/selfoss#1120]
Hi,

hope this hasn't been covered and rejected in another issue, but i did not find an open one at least.

I just added a new source and found that multiple sources were in an error state. 

Some with a 500 error (usually that comes and goes...) but also quite a few 404'd since the last time I added a feed.

Is there an easy way how this could be communicated to the user, so it would directly be visible if a feed 404s? Then you don't keep waiting for an update for months altough the site operator has decided to rename feed.xml to rss.xml or vice versa...

Maybe there could be a small red bubble with a number of failed feeds on the sources icon in the left panel? Similar to the red bubble next to "unread".

Alternatively there could be a toast, but that would mean to store whether it has been shown or show it everytime, which could be quite annoying if there are feeds which regularly and intermittently have 500s or so...

But maybe someone has an even better idea how to tackle this :-)

**TL;DR:**
Problem: i do not recognize directly if i have broken feeds, but it bugs me if i find out later a feed was broken for months
Possible solution: find a way to notify user about broken feeds that is not obtrusive, like showing a number of failed feeds next to the sources-icon.

Thanks for considering!
